metadata-ingestion/src/datahub/ingestion/source/bigquery_v2/bigquery_schema_gen.py

```diff
@@ -708,6 +722,8 @@ def gen_table_dataset_workunits(
             tags_to_add.extend(
                 [
                     make_tag_urn(f"""{k}:{v}""")
+                    if not v.startswith(ENCODED_TAG_PREFIX)
+                    else self.modified_base32decode(v)
```

Comment:
let's pull this out into a `make_tag_from_label(k, v)` helper method, and ideally add some tests for that helper